packages/react-components/react-badge/src/components/Badge/Badge.tsx

```diff
@@ -4,14 +4,19 @@ import { useBadgeStyles_unstable } from './useBadgeStyles';
 import { renderBadge_unstable } from './renderBadge';
 import type { BadgeProps } from './Badge.types';
 import type { ForwardRefComponent } from '@fluentui/react-utilities';
+import { useCustomStyleHooks_unstable } from '@fluentui/react-shared-contexts';
```

Comment:
_nit:_ Keep `import type` statements at the bottom

```suggestion
import { useCustomStyleHooks_unstable } from '@fluentui/react-shared-contexts';
import { renderBadge_unstable } from './renderBadge';
import type { BadgeProps } from './Badge.types';
import type { ForwardRefComponent } from '@fluentui/react-utilities';
```